ksql-engine/src/test/java/io/confluent/ksql/EndToEndEngineTestUtil.java

```diff
@@ -779,10 +786,42 @@ private static TopologyAndConfigs readTopologyFile(final String file, final Obje
     }
   }
 
-  static Stream<JsonTestCase> findTestCases(final Path dir) {
+  private static List<Path> getTests(final Path dir, final List<String> files) {
+    List<Path> filePaths = new ArrayList<>();
+    for (String file : files) {
+      filePaths.add(dir.resolve(file.trim()));
+    }
+
+    return filePaths;
+  }
+
+  /**
+   * Returns a list of files specified in the system property 'ksql.test.file'.
+   * The list may be specified as a comma-separated string.
+   */
+  public static List<String> getTestFilesParam() {
+    String propValue = System.getProperty(KSQL_TEST_FILES);
```

Comment:
maybe: `return Arrays.asList(System.getProperty(KSQL_TEST_FILES, "").trim().split(","));`

Reply:
That line will return an array of one empty item if ksql.test.files is empty. I'd rather return an empty array with zero elements. I simplify the code a little bit, though, to take advantage of the getProperty(var, default). Thanks.